components/node/eth/account_proof.go

```diff
@@ -14,7 +14,6 @@ import (
 	"github.com/ethereum/go-ethereum/ethdb/memorydb"
 	"github.com/ethereum/go-ethereum/rlp"
 	"github.com/ethereum/go-ethereum/trie"
-
```

Comment:
I think this line should not be removed, isn't it?

Reply:
Yes, it has been removed in the new version. This is the old version

Reply:
I mistook our organization's package for the current package😅 Thank you!